static/app/components/replays/actionCategory.tsx

```diff
@@ -3,68 +3,52 @@ import styled from '@emotion/styled';
 
 import Tooltip from 'sentry/components/tooltip';
 import {t} from 'sentry/locale';
-import {
-  BreadcrumbType,
-  BreadcrumbTypeHTTP,
-  BreadcrumbTypeNavigation,
-  RawCrumb,
-} from 'sentry/types/breadcrumbs';
-
-import {convertCrumbType} from '../events/interfaces/breadcrumbs/utils';
+import {BreadcrumbType, Crumb} from 'sentry/types/breadcrumbs';
 
 type Props = {
-  category: RawCrumb;
-  description?: BreadcrumbTypeNavigation | BreadcrumbTypeHTTP;
+  action: Crumb;
+};
+
+type actionCategoryInfo = {
```

Comment:
PascalCase type names, they are like classes, variables are snakeCase.

```suggestion
type ActionCategoryInfo = {
```